paddle/fluid/imperative/tracer.cc

```diff
@@ -47,6 +49,9 @@ void Tracer::TraceOp(const std::string& type, const NameVarBaseMap& ins,
                      const NameVarBaseMap& outs, framework::AttributeMap attrs,
                      const platform::Place& place, bool trace_backward) {
   VLOG(1) << "Trace Op: " << type;
+  if (FLAGS_use_mkldnn) {
+    attrs["use_mkldnn"] = true;
+  }
```

Comment:
LGTM for dygraph mode.
I don't see handling `use_mkldnn` in static mode, is it already done in other PR?

Reply:
If you mean running through executor, it's already done in executor.cc:185